subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/core_dependency_management.adoc

```diff
@@ -1,60 +1,60 @@
 [[dependency_management_in_gradle]]
-= Dependency management in Gradle
+= Dependency Management
 
-== What is dependency management?
+Software projects rarely work in isolation.
+Projects often rely on reusable functionality from libraries.
+Some projects organize unrelated functionality into separate parts of a modular system.
+Dependency management is an automated technique for declaring, resolving and using functionality required by a project.
 
-Software projects rarely work in isolation. In most cases, a project relies on reusable functionality in the form of libraries or is broken up into individual components to compose a modularized system. Dependency management is a technique for declaring, resolving and using dependencies required by the project in an automated fashion.
-
-[NOTE]
+[TIP]
 ====
-For a general overview on the terms used throughout the user guide, refer to <<dependency_management_terminology.adoc#dependency_management_terminology,Dependency Management Terminology>>.
+For an overview of dependency management terms, see <<dependency_management_terminology.adoc#dependency_management_terminology,Dependency Management Terminology>>.
 ====
 
 [[sec:dependency-mgmt-in-gradle]]
-== Dependency management in Gradle
+== Dependency Management in Gradle
 
-Gradle has built-in support for dependency management and lives up to the task of fulfilling typical scenarios encountered in modern software projects.
-We’ll explore the main concepts with the help of an example project.
-The illustration below should give you an rough overview on all the moving parts.
+Gradle has built-in support for dependency management.
+We’ll explore the main concepts with the help of the theoretical example project illustrated below:
```

Comment:
question: should we just remove the word example per @jbartok suggestion?
```suggestion
We’ll explore the main concepts with the help of the theoretical project illustrated below:
```

Reply:
Good point, I think "theoretical project" does enough work here on its own. Good example of a word we can eliminate with a bonus of eliminating confusion.